kolibri_exercise_perseus_plugin/downloadTranslations.js

```diff
@@ -0,0 +1,59 @@
+const {
+  localeFile,
+  crowdinLangCode,
+  localLocaleFile,
+  remoteLocaleFile,
+  kolibriProjectId,
+  downloadTranslations,
+  languages,
+} = require('./translations');
+const {
+  kolibriApiKey,
+} = require('../crowdinSecrets');
```

Comment:
Make sure to add this in the gitignore file!

Reply:
Already there.